### PR TITLE
Fix installation procedure in readme.

### PR DESCRIPTION
**Pull request by newPOPE**

### CODE REVIEW

**Comment by JanTvrdik** [comment]
:-1:

File: readme.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Installation
 
 ```
 # Clone repository
-git clone https://github.com/nette/web-addons.nette.org.git
+git clone https://github.com/nette/web-addons.nette.org.git addons.nette.org
 cd addons.nette.org
```
Comment (Majkl578) [inline readme.md]: You should fix this instead I guess.

Comment (JanTvrdik) [inline readme.md]: :+1: That seems more reasonable.

**Comment by newPOPE** [comment]
@JanTvrdik: :-1: says nothing. Explanation must be more detailed

**Comment by JanTvrdik** [comment]
> Explanation must be more detailed

The same is true for pull request description. I originally had no idea what is this supposed to fix.

**Comment by newPOPE** [comment]
So close or what?

**Comment by Majkl578** [comment]
Merged, but reverted afterwards when I noticed there are 3 commits (incl. 1+1 revert)... So cherry-picked the last of them only. Next time please squash the changes into one commit. :)

**Comment by newPOPE** [comment]
Ok, accepted. Thanks.

**Pull request merged**